.prettierrc.js

```diff
@@ -18,15 +18,22 @@ module.exports = {
   // See: https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/335#issuecomment-2801686781
   overrides: [
     {
-      // React web components use the Storybook React Tailwind config
-      files: ['./packages/design-system-react/src/**'],
+      // React web components and stories use the Storybook React Tailwind config
+      files: [
+        './packages/design-system-react/src/**',
+        './apps/storybook-react/stories/**',
+        './packages/design-tokens/stories/**',
```

Comment:
Adds linting to storybook stories in design tokens and storybook react

Reply:
Moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/683